javatests/arcs/android/e2e/testapp/StorageAccessService.kt

```diff
@@ -63,27 +64,15 @@ class StorageAccessService : LifecycleService() {
             ) as WriteSingletonHandle<TestEntity>
 
             singletonHandle.onReady {
-                scope.launch {
-                    when (action) {
-                        Action.SET -> {
-                            singletonHandle.store(
-                                TestEntity(
-                                    text = TestEntity.text,
-                                    number = TestEntity.number,
-                                    boolean = TestEntity.boolean
-                                )
-                            )
-                        }
-                        Action.CLEAR -> {
-                            singletonHandle.clear()
-                        }
-                    }
-
-                    singletonHandle.close()
-                    handleManager.close()
+                when (action) {
+                    Action.SET -> singletonHandle.store(
+                        TestEntity(TestEntity.text, TestEntity.number, TestEntity.boolean)
+                    )
+                    Action.CLEAR -> singletonHandle.clear()
                 }
+                singletonHandle.close()
+                runBlocking { handleManager.close() }  //~ scope.launch?
```

Comment:
What does this comment mean?

Reply:
Err, that's my "don't submit with this" marker... so it means I forgot to check for that before sending out :-/

The actual question here was whether the `scope.launch` enclosing the onReady logic was actually required. I suspect it's a hold-over from before handles and proxies were made single-threaded.